lib/utils/import-href.html

```diff
@@ -36,9 +36,9 @@
    *
    * @memberof Polymer
    * @param {string} href URL to document to load.
-   * @param {!Function=} onload Callback to notify when an import successfully
+   * @param {!function(!Event):void=} onload Callback to notify when an import successfully
```

Comment:
Can be `:void`

Reply:
again the callback is optional, it cannot.